rolling/distribution.yaml

```diff
@@ -2189,12 +2189,12 @@ repositories:
     doc:
       type: git
       url: https://github.com/open-rmf/rmf_battery.git
-      version: rolling
+      version: main
     release:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/rmf_battery-release.git
-      version: 0.1.0-1
+      version: 0.1.0-2
```

Comment:
Is this the version you expect? This kind of bump usually only happens when you re-run bloom but no new upstream version exists. It could be what you meant to do, just double checking.

Reply:
Thanks for the double check. Yes, initially we were using `rolling` branches for the `rolling` distribution but in the end using directly `main` makes more sense so we are slowly moving them all to `main`.

Reply:
@marcoag in the future you can open PRs to change the doc and source branches without bloom and skip the bloom revision.